### PR TITLE
Add: permanent services initial

### PR DESCRIPTION
**Pull request by dpaez**
This adds a new package for the permanent seeder services. Thanks to @tinchoz49

### CODE REVIEW

File: packages/services/.editorconfig [hunk 1]
```diff
@@ -0,0 +1,9 @@
+root = true
```
Comment (estebanprimost) [inline packages/services/.editorconfig]: Why not to have a single file on root?

File: packages/services/.gitignore [hunk 2]
```diff
@@ -0,0 +1,61 @@
+# Logs
```
Comment (estebanprimost) [inline packages/services/.gitignore:1]: Same as `.editorconfig` (single file on root)

**Review by estebanprimost — APPROVED** [review]
Seems good to me

File: packages/services/.editorconfig (hunk 1 above)
Comment (dpaez) [reply packages/services/.editorconfig]: yeah, this would be the best, also this would affect other packages too.

**Review by estebanprimost — APPROVED** [review]

**Pull request merged**